cluster/gce/config-default.sh

```diff
@@ -307,7 +307,7 @@ NODE_PROBLEM_DETECTOR_TAR_HASH="${NODE_PROBLEM_DETECTOR_TAR_HASH:-}"
 NODE_PROBLEM_DETECTOR_RELEASE_PATH="${NODE_PROBLEM_DETECTOR_RELEASE_PATH:-}"
 NODE_PROBLEM_DETECTOR_CUSTOM_FLAGS="${NODE_PROBLEM_DETECTOR_CUSTOM_FLAGS:-}"
 
-CNI_SHA1="${CNI_SHA1:-}"
```

Comment:
not sure if we need to keep setting the SHA1 hash vars and set SHA512 in parallel until the oldest node we support has inherited the SHA512 checking code.